test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/LocalSetup.md

```diff
@@ -0,0 +1,35 @@
+﻿## Software Installation
+
+Function apps for the test run in docker containers. Please install docker on the system.
+
+## Additional Resources
```

Comment:
How would it be additional Infra resources for better understanding ??

Reply:
Are you suggesting to add an explanation here?